picatrix/lib/framework.py

```diff
@@ -453,6 +454,24 @@ def _parse_line_string(line: str) -> List[str]:
   return arguments
 
 
+def picatrix_helper(function: Any) -> Any:
```

Comment:
Not sure how much you care about this (or the type checker for that matter), but if you want  to tell the type checker that the input and the output value are of the **same type**, using Any is insufficient, rather use type variables:

import typing

```
T = typing.TypeVar('T')

def id(x: T) -> T:
  return x
```

So if type checker sees:

```
y = id(1) + 1 # type check succeeds
z = id(1).append('foo') # type check should fail
```
It can validate that what the id function produces given an argument of type 'number' is of the same type, ie. number. Whereas with Any such checks are abandoned and the following is allowed:

```
def id(x: Any) -> Any:
  return [x,x,x]
```